Milad.css

```diff
@@ -0,0 +1,127 @@
+/* Reset some default styles */
+body, h1, h2, h3, p, ul, li {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: Comic sans MS, sans-serif;
+    background-color: #f0f0f0;
+    background-image: url(https://www.w3schools.com/css/lights600x400.jpg);
+  background-repeat: no-repeat;
+  background-size:  cover;
+  color: #FFFFFF;
+  height: 700px;
+  width: auto;
+  
+      
+      
+}
+
+
+header {
+    background-color: #FFC0CB;
+    color: #48D1CC;
+    text-align: center;
+    padding: 2rem 0;
+}
+
+nav {
+    background-color: rgba(0, 0, 0, 0.7);
+    color: #FFFFFF;
+    padding: 10px 0;
+}
+
+nav ul {
+    list-style: none;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0;
+    padding: 0;
+}
+
+nav li {
+    margin: 0 20px;
+}
+
+nav a {
+    text-decoration: none;
+    color: #fff;
+    transition: color 0.3s;
+}
+
+nav a:hover {
+    color: #00f;
+    text-decoration: none;
+}
+
+.dropdown {
+    position: relative;
+}
+
+.dropdown-content {
+    display: none;
+    position: absolute;
+    background-image: url(https://www.w3schools.com/css/lights600x400.jpg);
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+    border-radius: 5px;
+    padding: 50px;
+    top: 100%;
+    left: 0;
+    z-index: 1;
+}
+
+.dropdown:hover .dropdown-content {
+    display: block;
+}
+
+.projects-list {
+    list-style: none;
+    margin-top: 60px;
+    padding: 0;
+}
+
+footer {
+    text-align: right;
+    padding: 1rem 0;
+    background-color: lightblue;
+    color: red;
+    padding-right: 15px;
+    margin-bottom: 5px;
+}
```

Comment:
I would prefer to use rem or em units instead of px because px is an absolute unit size, which can hinder your website's responsiveness.

To learn more, consider reading articles linked below.


https://www.freecodecamp.org/news/why-use-rem-to-set-font-size-in-css/#:~:text=For%20styling%20a%20webpage%2C%20we,the%20size%20of%20the%20screen.